File: JavaProto/app/src/main/java/com/example/javaproto/Character.java
package com.example.javaproto;

import android.graphics.Canvas;
import android.graphics.Color;
import android.graphics.Paint;
import android.graphics.RectF;

public class Character extends RectF{
    private int colDur=0;
    boolean isOut=false;
    Paint paint3 = new Paint();
    public int getColDur() {
        return colDur;
    }
    public void setColDur(int colDur) {
        this.colDur = colDur;
    }
    public Character(float left, float top, float right, float bottom) {
        super(left, top, right, bottom);
    }

}

File: MyApplication8/app/src/main/java/com/example/myapplication/Character.kt
package com.example.myapplication

import android.graphics.Paint
import android.graphics.RectF

class Character(left: Float=0f, top: Float=0f, right: Float=0f, bottom: Float=0f) : RectF(left,top,right,bottom) {
    var colDur: Int = 0
    var isOut: Boolean = false
    var paint3: Paint = Paint()
}